react/table/src/state/filter.types.js

```diff
@@ -0,0 +1,4 @@
+export const ADD_FILTER = 'addFilter';
```

Comment:
what about adding kind of namespace to all action types? if the developer chooses to move the state stuff to redux, it would be very useful (and we could provide a redux option from the generator side as well)

Reply:
👍 added